metron-platform/metron-common/src/test/java/org/apache/metron/common/utils/StellarExecutor.java

```diff
@@ -0,0 +1,84 @@
+package org.apache.metron.common.utils;
```

Comment:
I have also taken the executor functions from StellarTest and made them into a util class.  There is some overlap here.

https://github.com/apache/incubator-metron/blob/master/metron-platform/metron-common/src/test/java/org/apache/metron/common/utils/StellarProcessorUtils.java